requirements/dev.txt

```diff
@@ -5,57 +5,55 @@
 #    make upgrade
 #
 appdirs==1.4.4            # via fs, virtualenv
-asgiref==3.3.1            # via django
 astroid==2.4.2            # via pylint, pylint-celery
 attrs==20.3.0             # via pytest
-bleach==3.2.1             # via readme-renderer
+bleach==3.3.0             # via readme-renderer
 boto==2.49.0              # via -r requirements/base.in
 certifi==2020.12.5        # via requests
-cffi==1.14.4              # via cryptography
-chardet==4.0.0            # via pysrt, requests
+cffi==1.14.5              # via cryptography
+chardet==4.0.0            # via diff-cover, pysrt, requests
 click-log==0.3.2          # via edx-lint
 click==7.1.2              # via click-log, code-annotations, edx-lint, pip-tools
-code-annotations==0.10.2  # via edx-toggles
+code-annotations==1.1.0   # via edx-lint, edx-toggles
 colorama==0.4.4           # via twine
-coverage==5.3.1           # via -r requirements/test.in, coveralls, pytest-cov
+coverage==5.4             # via -r requirements/test.in, coveralls, pytest-cov
 coveralls==3.0.0          # via -r requirements/travis.in
-cryptography==3.3.1       # via django-fernet-fields, pyjwt, secretstorage
+cryptography==3.4.5       # via django-fernet-fields, pyjwt
 ddt==1.4.1                # via -r requirements/test.in
-diff-cover==4.1.1         # via -r requirements/dev.in
+diff-cover==4.2.1         # via -r requirements/dev.in
 distlib==0.3.1            # via virtualenv
 django-crum==0.7.9        # via edx-django-utils, edx-toggles
 django-fernet-fields==0.6  # via -r requirements/base.in
 django-model-utils==4.1.1  # via -r requirements/base.in
 django-storages==1.11.1   # via -r requirements/base.in
-django-waffle==2.0.0      # via edx-django-utils, edx-drf-extensions, edx-toggles
-django==3.1.5             # via -r requirements/base.in, code-annotations, django-crum, django-fernet-fields, django-model-utils, django-storages, djangorestframework, drf-jwt, edx-django-utils, edx-drf-extensions, edx-toggles, rest-condition
+django-waffle==2.1.0      # via edx-django-utils, edx-drf-extensions, edx-toggles
+django==2.2.18            # via -c https://raw.githubusercontent.com/edx/edx-lint/master/edx_lint/files/common_constraints.txt, -r requirements/base.in, code-annotations, django-crum, django-fernet-fields, django-model-utils, django-storages, djangorestframework, drf-jwt, edx-django-utils, edx-drf-extensions, edx-lint, edx-toggles, rest-condition
```

Comment:
django is downgrading here. Is this the expected behavior?

Reply:
We are staying on Django 2.2 org-wide in edX as it is the `LTS` version though testing for 3.0 and 3.1 is added but not upgraded to those versions yet. If you don't want this to downgrade, we can discard this PR.